src/app/core/toolbar/toolbar.component.spec.ts

```diff
@@ -12,9 +13,11 @@ import { ToolbarComponent } from './toolbar.component'
 describe('ToolbarComponent', () => {
   let component: ToolbarComponent
   let fixture: ComponentFixture<ToolbarComponent>
-
+  let logoutSpy: jasmine.Spy
   beforeEach(async () => {
     const routerSpy = jasmine.createSpyObj('Router', ['navigate'])
+    const authService = jasmine.createSpyObj('AuthService', ['logout'])
+    logoutSpy = authService.logout
```

Comment:
what is this used for? You can use authService.logout on line 54.

Reply:
done